### DISCUSSION TITLE
Controls with v-bind not working

### DISCUSSION THREAD

**Discussion by victorlmneves** [histoire-dev/histoire#163] [Q&A]
I have components where I pass the data/props using v-bind, but when I try to override the initial data nothing happens on the component. I can only see it if I pass each prop separately. Is there a way of making the controls work with v-bind?

<img width="560" alt="image" src="https://user-images.githubusercontent.com/17341789/177520389-80f89778-da1f-4175-8a77-f5c1572dab35.png">

Here I set the appearance to secondary and disabled and nothing happened. Also, the text should be visible because the `isTextVisible` is false.
![image](https://user-images.githubusercontent.com/17341789/177520951-3a6a564d-d271-4b78-b20c-76b9b696ed58.png)

Thanks

**Comment by Akryum** [comment]
Hey, could you create an issue about that? Thanks!
